Makefile

```diff
@@ -50,25 +50,28 @@ binary: build
 	$(DOCKER_RUN_DOCKER) hack/make.sh binary
 
 cross: build
-	$(DOCKER_RUN_DOCKER) hack/make.sh binary cross
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary binary cross
 
 deb: build
-	$(DOCKER_RUN_DOCKER) hack/make.sh binary build-deb
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary build-deb
+
+dynbinary: build
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary
```

Comment:
We already run distro specific tests and builds.  But doesn't this docker run docker in docker?